Feature/code system generation

Censacrof opened the pull request:
Work related to #32 .

I created a script that is able to get **all** coding systems and their translations from the EDQM API. I used it to generate **.fhs** coding system files for **all** EDQM concepts under **input/fsh/coding-system/edqm**. I used SUSHI to generate the FHIR resources, and it gave no error, so at least from the syntax point of view should be ok. 

Please, let me know if I need to change / improve something. I also plan on doing a similar script for the SMS substances.

costateixeira commented:
@Censacrof  Your build is failing but the FHIR content is OK, it fails because you don't have the github pages activated. If you do enable them (Settings, pages, enable publication from the gh-pages branch), your branch will always as an IG in https://<owner>.github.io/<repo> 
or, if it's a branch, https://<owner>.github.io/<repo>branches/<branch>
That way you can check if the build is producing the expected content, and you can check the QA page.

Censacrof commented:
> @Censacrof Your build is failing but the FHIR content is OK, it fails because you don't have the github pages activated. If you do enable them (Settings, pages, enable publication from the gh-pages branch), your branch will always as an IG in https://.github.io/ or, if it's a branch, https://.github.io/branches/ That way you can check if the build is producing the expected content, and you can check the QA page.

Thanks you Jose, I will try that.

Censacrof commented:
@costateixeira I enabled GitHub Pages but it fails anyway on the "Deploy Candidate" job. Tomorrow I will look into it.

```
Run JamesIves/github-pages-deploy-action@4.1.4

    GitHub Pages Deploy Action 🚀

    🚀 Getting Started Guide: https://github.com/marketplace/actions/deploy-to-github-pages
    ❓ Discussions / Q&A: https://github.com/JamesIves/github-pages-deploy-action/discussions
    🔧 Report a Bug: https://github.com/JamesIves/github-pages-deploy-action/issues

    📣 Maintained by James Ives: https://jamesiv.es
    💖 Support: https://github.com/sponsors/JamesIves
Checking configuration and starting deployment… 🚦
Deploying using Deploy Token… 🔑
Configuring git…
/usr/bin/git config user.name Censacrof
/usr/bin/git config user.email Censacrof@users.noreply.github.com
/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
/usr/bin/git remote rm origin
/usr/bin/git remote add origin ***github.com/hl7-eu/unicom-ig.git
Git configured… 🔧
Starting to commit changes…
/usr/bin/git ls-remote --heads ***github.com/hl7-eu/unicom-ig.git refs/heads/gh-pages
8e1[9](https://github.com/hl7-eu/unicom-ig/actions/runs/3601945501/jobs/6076759972#step:9:10)28085c9f5f34e2bc051133acd21ab396a11c	refs/heads/gh-pages
Creating worktree…
/usr/bin/git fetch --no-recurse-submodules --depth=1 origin gh-pages
From https://github.com/hl7-eu/unicom-ig
 * branch            gh-pages   -> FETCH_HEAD
 * [new branch]      gh-pages   -> origin/gh-pages
/usr/bin/git worktree add --no-checkout --detach github-pages-deploy-action-temp-deployment-folder
Preparing worktree (detached HEAD 2701f34)
/usr/bin/git checkout -B gh-pages origin/gh-pages
Previous HEAD position was 2701f34 Merge c384970dcfb817078b2c7ae58c5b2a5c276d75d2 into 482c224e5e9b6f2468d84[10](https://github.com/hl7-eu/unicom-ig/actions/runs/3601945501/jobs/6076759972#step:9:11)ccaffdf2006606272
Switched to a new branch 'gh-pages'
branch 'gh-pages' set up to track 'origin/gh-pages'.
Creating target folder if it doesn't already exist… 📌
/usr/bin/rsync -q -av --checksum --progress /home/runner/work/unicom-ig/unicom-ig/output/. github-pages-deploy-action-temp-deployment-folder/branches/feature/code-system-generation --exclude .ssh --exclude .git --exclude .github
Checking if there are files to commit…
/usr/bin/git add --all .
/usr/bin/git checkout -b github-pages-deploy-action/62lr9j3k8
Switched to a new branch 'github-pages-deploy-action/62lr9j3k8'
/usr/bin/git commit -m Deploy candidate branch --quiet --no-verify
/usr/bin/git push --force ***github.com/hl7-eu/unicom-ig.git github-pages-deploy-action/62lr9j3k8:gh-pages
remote: Permission to hl7-eu/unicom-ig.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/hl7-eu/unicom-ig.git/': The requested URL returned error: 403
Running post deployment cleanup jobs… 🗑️
/usr/bin/git checkout -B github-pages-deploy-action/62lr9j3k8
Reset branch 'github-pages-deploy-action/62lr9j3k8'
/usr/bin/git worktree remove github-pages-deploy-action-temp-deployment-folder --force
Error: The deploy step encountered an error: The process '/usr/bin/git' failed with exit code 128 ❌
Deployment failed! ❌

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-[11](https://github.com/hl7-eu/unicom-ig/actions/runs/3601945501/jobs/6076759972#step:9:12)-github-actions-deprecating-save-state-and-set-output-commands/
```
